server/migration/1623276130329-FormatPhoneNumbers.ts

```diff
@@ -0,0 +1,57 @@
+import {
+  MigrationInterface,
+  QueryRunner,
+} from 'typeorm';
+import { formatNumber } from 'libphonenumber-js';
+
+export class FormatPhoneNumbers1623276130329 implements MigrationInterface {
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await updatePhoneNumbers(queryRunner, phone => formatNumber(phone, 'US', 'NATIONAL'));
+  }
+
+  public async down(queryRunner: QueryRunner): Promise<void> {
+    await updatePhoneNumbers(queryRunner, phone => {
+      const phoneDigits = phone.replace(/\D/g, '');
+      return (phoneDigits.length === 10)
+        ? `${phoneDigits.slice(0, 3)}-${phoneDigits.slice(3, 6)}-${phoneDigits.slice(6)}`
+        : phone;
+    });
+  }
+
+}
+
+async function updatePhoneNumbers(queryRunner: QueryRunner, formatter: (phone: string) => string) {
```

Comment:
Should this also do Roster? RosterHistory?

Reply:
There's no official phone column or type for Roster/RosterHistory, but it's probably a good idea to do something similar to the muster stuff with a sort of fuzzy search for one.